### ISSUE TITLE
Sticker API not working

### ISSUE THREAD

**Issue by allanmelo** [open-wa/wa-automate-nodejs#1298]
**Describe the bug**
All messages sent with the "sendImageAsSticker" function have stopped working properly since the update to version 3.2.5. A downgrade to version 3.2.4 makes this function ok for use.

**Steps to Reproduce**

```javascript
if (message.isMedia && message.type === 'image') {
    const mediaData = await decryptMedia(message, uaOverride)
    const imageBase64 = `data:${mimetype};base64,${mediaData.toString('base64')}`
    await client.sendImageAsSticker(from, imageBase64)
}
```

**Expected behavior**
Send image as sticker.

**Actual behavior**
Did not reply any image. Show "undefined undefined" on console. Didn't ask to show any kind of error on console.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.2.6](https://github.com/open-wa/wa-automate-nodejs/compare/3.2.5...3.2.6) (2021-02-04)

- build(deps): bump puppeteer from 5.5.0 to 6.0.0 [`#1295`](https://github.com/open-wa/wa-automate-nodejs/pull/1295)
- build(deps): bump puppeteer-extra-plugin-stealth from 2.6.7 to 2.7.3 [`#1294`](https://github.com/open-wa/wa-automate-nodejs/pull/1294)
- Release 3.2.6 [`4a082f3`](https://github.com/open-wa/wa-automate-nodejs/commit/4a082f36838b7932bff8e418c8afb63d9c782666)
- fix: #1263 [`6457822`](https://github.com/open-wa/wa-automate-nodejs/commit/6457822969517c5a47d9fee698d203e8b7a8efeb)
- fix: `sendText` error message fixed [`848a60a`](https://github.com/open-wa/wa-automate-nodejs/commit/848a60ac732cc8232b986b7ef0f4ce9c751c7a8e)
- fix: #1298 [`bc36422`](https://github.com/open-wa/wa-automate-nodejs/commit/bc36422ef58548f8eccb1054241e00d40703e59b)

**Comment by smashah** [comment]
@allanmelo fixed in [3.2.6](https://github.com/open-wa/wa-automate-nodejs/compare/3.2.5...3.2.6)
